core_backend/app/llm_call/process_input.py

```diff
@@ -76,7 +76,7 @@ async def _identify_language(
         IdentifiedLanguage, llm_identified_lang, IdentifiedLanguage.UNSUPPORTED
     )
     question.original_language = identified_lang
-    response.debug_info["original_language"] = identified_lang
+    response.debug_info["original_language"] = identified_lang.value
```

Comment:
@amiraliemami  I want to add this to the hotfix! I can add these commits but in case you can make them that'd also be great.